app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,18 @@
+module DeviseHelper
+  def devise_error_messages!
```

Comment:
remove this file 

add this gem - https://github.com/rails/dynamic_form
and then just put = f.error_messages